src/Tarjan.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Graphp\Algorithms;
+
+use Fhaculty\Graph\Exception\InvalidArgumentException;
+use Fhaculty\Graph\Graph;
+use Fhaculty\Graph\Set\Vertices;
+use Fhaculty\Graph\Vertex;
+
+/**
+ * @see http://github.com/Trismegiste/Mondrian/blob/master/Graph/Tarjan.php
+ */
+class Tarjan extends BaseGraph
```

Comment:
The `BaseGraph` will be deprecated, see also #5. The `Graph` should be passed to your *main method* instead.